panoptes_aggregation/reducers/subject_difficulty_reducer.py

```diff
@@ -0,0 +1,38 @@
+'''
+Subject Gold Standard Reducer for difficulty calculation
+---------------------------------------------------------
+This module provides functions to reduce the gold standard task extracts
+to determine a `difficulty' score per subject (defined as the fraction
+of succesful classification by all users for that subject)
+
+See also: tess_gold_standard_reducer
+'''
+from .reducer_wrapper import reducer_wrapper
+import numpy as np
+
+
+@reducer_wrapper()
+def subject_difficulty_reducer(extracts):
+    '''
+    Calculate the difficulty of a gold standard TESS subject
+
+    Parameters
+    ----------
+    extracts : list
+        The list of extracted data including the feedback metadata
+
+    Returns
+    -------
+    output : dict
+        A dictinary with one key `difficulty` that is a list with the fraction of volunteers who
+        successfully found each gold standard entry in a subject.
+    '''
+
+    feedback_data = [extracti['feedback'] for extracti in extracts if extracti['feedback'] is not None]
```

Comment:
What is `if extracti['feedback'] is not None` catching here?  I think in the cases where `feedback` is `None` the key does not exist at all (see line 63 of the extractors/utilities.py file).  Shouldn't this check be `if 'feedback' in extracti` to check for the key?

Reply:
This was a mistake on our end. It's supposed to be checking if extracti['feedback'] even exists. If the feedback info is not passed in for a classification (if a subject has no gold standard), the reducer should ignore that classification. This has been changed to 
```python
    feedback_data = [extracti['feedback'] for extracti in extracts if 'feedback' in extracti.keys()]
```